packages/core/src/__tests__/overall.test.ts

```diff
@@ -67,17 +67,18 @@ describe("Determinism", () => {
     const stateSample3Opt = resSample3Opt.value;
     const svgSample3Opt = await render(stateSample3Opt);
 
-    expect(svgSample1NotOpt).not.toBe(svgSample1Opt); // optimization does something
-    expect(svgSample1NotOpt).not.toBe(svgSample2NotOpt); // resampling is different from initial sampling
-    expect(svgSample1NotOpt).not.toBe(svgSample2Opt); // optimizing resample doesn't get us back
```

Comment:
Why did you remove these comments?

Reply:
@joshsunshine I didn't remove them; I just reorganized the assertions and prefaced each group with a comment that describes the whole group, rather than putting a comment at the end of every assertion line.